common/dynamicconfig/constants.go

```diff
@@ -627,16 +627,14 @@ const (
 	ExecutionsScannerEnabled = "worker.executionsScannerEnabled"
 	// HistoryScannerDataMinAge indicates the history scanner cleanup minimum age.
 	HistoryScannerDataMinAge = "worker.historyScannerDataMinAge"
-	// WorkerBatcherMaxConcurrentActivityExecutionSize indicates worker batcher max concurrent activity execution size
-	WorkerBatcherMaxConcurrentActivityExecutionSize = "worker.BatcherMaxConcurrentActivityExecutionSize"
-	// WorkerBatcherMaxConcurrentWorkflowTaskExecutionSize indicates worker batcher max concurrent workflow execution size
-	WorkerBatcherMaxConcurrentWorkflowTaskExecutionSize = "worker.BatcherMaxConcurrentWorkflowTaskExecutionSize"
-	// WorkerBatcherMaxConcurrentActivityTaskPollers indicates worker batcher max concurrent activity pollers
-	WorkerBatcherMaxConcurrentActivityTaskPollers = "worker.BatcherMaxConcurrentActivityTaskPollers"
-	// WorkerBatcherMaxConcurrentWorkflowTaskPollers indicates worker batcher max concurrent workflow pollers
-	WorkerBatcherMaxConcurrentWorkflowTaskPollers = "worker.BatcherMaxConcurrentWorkflowTaskPollers"
 	// EnableBatcher decides whether start batcher in our worker
 	EnableBatcher = "worker.enableBatcher"
+	// EnableBatcherPerNamespaceWorker decides whether start batcher worker per namespace
+	EnableBatcherPerNamespaceWorker = "worker.enableBatcherPerNamespaceWorker"
+	// BatcherRPS controls number the rps of batch operations
+	BatcherRPS = "worker.batcherRPS"
```

Comment:
We probably should thinking about rps limit for worker as a whole. For example, how to prevent many namespace start batch operation that overwhelm the system.

Reply:
Yes. I should discuss this on batcher and scheduler. If necessary, I will send another PR for the change.